manifests/f/FutureSight/Bingo/1.3.0/FutureSight.Bingo.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FutureSight.Bingo
+PackageVersion: 1.3.0
+Installers:
+- InstallerUrl: https://download1640.mediafire.com/oz35hmahfg8gJckIbNivpo7RHfYSu7nT6oPonwUVK26yJUEUvpcd55rNtkFifr4kmK8FCbNA9do0_NX8gVF8DRmB0LALjk8/eaq7ak43r3pku5k/Bingo_1-3-0.zip"           id="downloadButton
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Bingo_1-3-0\Bingo.exe
+  InstallerSha256: 3022A67931F09FFDD75D35000D19BAE9885190D00C0A51B51621C7B06B97687F

```

```suggestion
```suggestion
- InstallerUrl: https://download1640.mediafire.com/zd3zyp82cufg2SUBmXrsYHA6tE32pkoypg4jsmrQttAnlrv0TI8-9ZnY1FzJS4-pYUizT9cIOPreC8--swFLbi9Vkwmypi3AxWrTC8__VGSAqbJBdLkUruCj_ZAU35Po7eBELHiapqqVHMRwEz6rzHzW9O3Bxcf9oA2AVctgqp2U/eaq7ak43r3pku5k/Bingo_1-3-0.zip
  Architecture: x86
  InstallerType: zip
  NestedInstallerType: exe
  NestedInstallerFiles:
  - RelativeFilePath: Bingo_1-3-0\Bingo.exe
  InstallerSha256: 9E581D70F5CEC3BBC40D89FCBDA14E7F83C11E3C7229273172B254602612B625
```